src/promptflow/promptflow/batch/_base_executor_proxy.py

```diff
@@ -39,3 +42,33 @@ async def exec_aggregation_async(
     ) -> AggregationResult:
         """Execute aggregation nodes"""
         raise NotImplementedError()
+
+
+class APIBasedExecutorProxy(AbstractExecutorProxy):
```

Comment:
why executorProxy is under promptflow.batch not promptflow.executor?

Reply:
`promptflow.executor` currently refers more to the executor of Python. `ExecutorProxy` can only be used in batch mode, so it is more suitable under batch folder.